mantras.py

```diff
@@ -0,0 +1,46 @@
+import discord
+import logging
+from discord.utils import get
+from channels import REPETITIONS
+from roles import HIVE_MXTRESS
+
+class Mantras():
+
+    current_mantra = ""
+
+    async def load_mantra(self):
+        if Mantras.current_mantra != "": return True #No need to load a mantra if it's already there.
+        with open("data/current_mantra.txt", "r") as mantra_file:
+            Mantras.current_mantra = mantra_file.readline()
+            self.LOGGER.info(f"Mantra loaded from file: {Mantras.current_mantra}")
+            mantra_file.close()
```

Comment:
Throws an exception if `current_mantra.txt` doesn't already exist. Although it theoretically should be rare in prod, would still suggest adding some code to create the file if it doesn't already exist on disk.